pkg/backup/providers/s3/s3_clientwrapper.go

```diff
@@ -0,0 +1,90 @@
+package s3
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws/session"
```

Comment:
Imported package grouping

Reply:
goimports-reviser is a good tool to do "Imported package grouping" automatically